runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -135,6 +137,13 @@
 		final JMenuItem resetOthers = new JMenuItem("Reset others");
 		resetOthers.addActionListener(e -> xpTrackerPlugin.resetOtherSkillState(skill));
 
+		// ProgressWrapper containing SkillIcon and ProgressBar
+		final JPanel progressWrapper = new JPanel();
```

Comment:
This can remain on L211 since it's not used until then

Reply:
I brought this up here as we add an action listener for in a couple lines lower. I felt that was more appropriate than moving the listener lower down.